2020/CG-05-12.md

```diff
@@ -39,4 +39,177 @@ Installation is required, see the calendar invite.
 
 ## Meeting Notes
 
-To be added after the meeting.
+### Opening, welcome and roll call
+
+#### Opening of the meeting
+
+#### Introduction of attendees
+
+* Adam Klein
+* Alex Crichton
+* Alon Zakai
+* Alshamma
+* Andreas Rossberg
+* Arun Purushan
+* Arun Purushan
+* Ben Smith
+* Conrad Watt
+* Deepti Gandluri
+* Derek Schuff
+* Emanuel Ziegler
+* Francis McCabe
+* Gergely Buday
+* Heejin Ahn
+* Ingvar Stepanyan
+* Jakob Kummerow
+* JP Sugarbroad
+* Keith Miller
+* Lars Hansen
+* Luke Wagner
+* Manos Koukoutos
+* Mingqui Sun
+* ms2ger
+* Nick Fitzgerald
+* Paolo Severini
+* Pat Hickey
+* Paul Dworzanski
+* Petr Penzin
+* Rick
+* Ross Tate
+* Ryan Hunt
+* Sabine
+* Sven Sauleau
+* Thomas Lively
+* Tobias Tebbi
+* Wouter van Oortmersson
+* Zalim
+* Zhi An Ng
+
+
+### Find volunteers for note taking (acting chair to volunteer)
+
+### Adoption of the agenda
+
+Thomas Seconds
+
+### Proposals and discussions
+
+#### Review of action items from prior meeting.
+
+#### Poll: Propose WebAssembly BigInt<->i64 conversion in JS API for phase 4 ([spec PR](https://github.com/WebAssembly/spec/pull/707)).
+
+<TODO: SS To add slides>
+
+SS: Only thing missing for phase 4 is consensus from CG
+
+SS: Only thing to mention is no implementation in reference interpreter as this is a JS AP”I feature
+
+POLL: WebAssembly BigInt<->i64 conversion in JS API for phase 4
+
+| SF | F | N | A | SA |
+| - | - | - | - | - |
+| 17 | 10 | 2 | 0 | 0 |
+
+
+#### Discuss global ref.func initializers complicating validation ([reference-types issue#76](https://github.com/WebAssembly/reference-types/issues/76))
+
+https://docs.google.com/presentation/d/1EOwIRSpW3kJlSdiR529rIaXXfvzIP221LK5qv8oeGbE/edit?usp=sharing
+
+AR: 4th option would be to use element section - plug in another kind of section that plugs in ref declarations, not sure that’s worth doing either
```

Comment:
I dunno if it's common practice, but it seems like it might be worthwhile to copy the 3 options into these notes so that the following conversation is clear w/o reference to the slides.

Reply:
We don't normally do this, since often it's hard to represent the content of the slides in the notes format. Though in this case it's not too hard, and it will make it easier to read.